pkg/views/gitprovider/select.go

```diff
@@ -62,6 +63,9 @@ func GitProviderCreationView(ctx context.Context, apiClient *apiclient.APIClient
 		}
 	}
 
+	var selectedSigningMethod string
```

Comment:
When running `dtn gp update`, the signing method is always set to `None` even though I set `gpg` on the provider. These 2 values need to be fetched from the `gitProviderAddView` if they are set.

Reply:
@bryans-go please provide a video to show that this works because it doesn't for me.

Reply:
https://github.com/user-attachments/assets/16c5ad1a-9599-4489-976f-ce44654a4e90

The value is preselected correctly now but the update does not go through when I want to set the signing method to `None` (was `GPG` before). Please make sure the git provider updates correctly in this scenario as well.